Doc/Bib/README.md

```diff
@@ -0,0 +1,109 @@
+(e.g. theses).
+
+- `NonVerif.MR`: Things not yet verified, same format as GapCite.MR.
+
+- `NonVerif.NonMR`: Things not yet verified, same format as `GapCite.notyet`
+
+
+## Checking MathSciNet for new GAP citations
+
+Steps described in this section require MathSciNet subscription and
+should be performed from the computer having IP address agreed with
+MathSciNet (arrangements made in St Andrews).
+
+
+### Identifying new GAP citations in MathSciNet
+
+To check MathSciNet for new GAP citations read `updatebib.g` into GAP.
+Running `updatebib.g` will produce two files:
+
+- `tobeadded.txt`: This file has the same format as `GapCite.MR` and
+lists publications citing GAP which should be examined and after that
+added either to `GapCite.MR` or to `GapIgnore.MR`.
+
+- `suggested.txt`: This file contains suggestions to "move" certain
+entries from `GapCite.notyet` and `GapNonMR.bib` to `GapCite.MR`.
+All suggestions, including those which do not match the publication
+listed in the GAP bibliography, should be carefully examined before
+any changes.
+
+Note that `updatebib.g` is not a complete solution for updating GAP
+bibliography. It searches for occurrences of the substring `www.gap`
+and `gap-system.org` in citations (this covers both old and current
+addresses of the GAP website), but it does not cover publications
+citing GAP without its website or referring to it only in the text;
+finally, it covers only MathSciNet and does not look into other
+bibliography databases. Therefore, manual search still should be used
+to discover more GAP citations. The  `SearchMathSciNetForUpdates`
+function from `updatebib.g` may be helpful in this direction since it
+performs more broad search in the MathSciNet, dropping some more strict
+limitations.
+
+### Retrieving BibTeX entries for GAP citations from MathSciNet
+
+After the source files of the GAP bibliography are updated, the script
+`makegapbib` uses `GapCite.MR`, `GapCite.notyet` and `GapNonMR.bib`
+to produce `gap-published.bib` (this requires subscription to MathSciNet,
+which St Andrews has). The advantage of this approach is that MathSciNet
+gives us good BiBTeX entries (no need to look up journal names or
+diacritic characters) and their updates, and MR numbers we can link to.
+It also makes it easier to add entries as only the MR number is needed.
+
+At the end of its work `makegapbib` will also display error messages
+reporting MR numbers whose BibTeX record it failed to fetch from
+MathSciNet -- these should be investigated since they may point out on
+some inconsistencies in our data.
+
+
+## Updating GAP bibliography webpages
+
+The next steps do not require MathSciNet subscription and can be
+performed elsewhere.
+
+Running `convbib.g` in GAP produces the following files, which are
+linked from the main GAP bibliography web page `bib.html`:
+
+- `gap-published.pdf`: a nice PDF bibliography from `gap-published.bib`
+(using further helper files `gapbib.tex` and `gap-head.bib`).
```

Comment:
I noticed that we actually have a link to this file in the HTML code (in `Doc/Bib/bib.html`), but it is not in the repository. I'd prefer not to have PDFs in there anyway, but that means we have a broken link...

Reply:
My preferred solution would be to remove the link to gap-published.pdf

Reply:
Oh I see, this PR adds that file. Hrm, I am not thrilled about this.

To be totally honest: I see little value in the whole of `Doc/Bib/`, and while I am personally willing to accommodate it to a degree, as some people apparently do see some value in it, I don't want it to complicate other things, nor blow up the repository size unduly.

Reply:
I have completely opposite opinion. It is used in various grants applications, to start with, and that's not the only reason. 

The other thing however is that the older setup is quite dated, and I'd do it differently now. Perhaps rewrite all in Python (and I have already Python code to put them on a map which is a nice visualisation.

It's not clear to me how do many people use each of the three formats:
- BibTeX source,
- Nicer readable HTML version,
- PDF file

i.e. is someone really downloading 187 pages PDF? So, we can only provide bibtex and html, and then avoid storing the binary PDF file in the repository.

Reply:
I totally respect that you and others value Doc/Bib/ which is why I a not suggesting to remove it (and who knows, I myself might one day be grateful it's here!).

My main concern is the PDF. If we can agree to remove that (we can still offer the .bib and .tex source for it, of course), I am totally happy to keep Doc/Bib/ in. At least as long as somebody maintains it, which you are evidently doing. Thanks for that, too, BTW :-)